services/capture/src/main/java/ca/bc/gov/tno/services/capture/config/CaptureConfig.java

```diff
@@ -189,4 +203,32 @@ public boolean getCaptureSuccess() {
   public void setCaptureSuccess(boolean success) {
     this.captureSuccess = success;
   }
+
+  /**
+   * @return String return the command for this capture process
+   */
+  public String getRunningCommand() {
+    return runningNow;
+  }
+
+  /**
+   * @param url the command for this capture process to set
```

Comment:
comment doesn't seem to match the variable?

Reply:
Fixed it